src/resolvers/user_management.spec.js

```diff
@@ -83,97 +95,165 @@ describe('currentUser', () => {
+    });
 
-    describe('with a valid email but incorrect password', () => {
+    describe("with a valid email but incorrect password", () => {
       it('responds with "Incorrect email address or password."', async () => {
         await expect(
-          request(host, mutation({
-            email: 'test@example.org',
-            password: 'wrong'
-          }))
-        ).rejects.toThrow('Incorrect email address or password.')
-      })
-    })
-
-    describe('with a non-existing email', () => {
+          request(
+            host,
+            mutation({
+              email: "test@example.org",
+              password: "wrong"
+            })
+          )
+        ).rejects.toThrow("Incorrect email address or password.");
+      });
+    });
+
+    describe("with a non-existing email", () => {
       it('responds with "Incorrect email address or password."', async () => {
         await expect(
-          request(host, mutation({
-            email: 'non-existent@example.org',
-            password: 'wrong'
-          }))
-        ).rejects.toThrow('Incorrect email address or password.')
-      })
-    })
-  })
-})
+          request(
+            host,
+            mutation({
+              email: "non-existent@example.org",
+              password: "wrong"
+            })
+          )
+        ).rejects.toThrow("Incorrect email address or password.");
+      });
+    });
+  });
+});
+
+describe("change password", () => {
+  let headers;
+  let client;
+
+  beforeEach(async () => {
+    headers = await login({ email: "test@example.org", password: "1234" });
+    client = new GraphQLClient(host, { headers });
+  });
+
+  const mutation = params => {
+    const { oldPassword, newPassword } = params;
+    return `
+      mutation {
+        changePassword(oldPassword:"${oldPassword}", newPassword:"${newPassword}")
+      }`;
+  };
+
+  describe("should be authenticated before changing password", () => {
+    it('should throw not "Not Authorised!', async () => {
+      await expect(
+        request(
+          host,
+          mutation({
+            oldPassword: "1234",
+            newPassword: "1234"
+          })
+        )
+      ).rejects.toThrow("Not Authorised!");
+    });
+  });
+
+  describe("old and new password should not match", () => {
+    it('responds with "Old password and New password should not be same"', async () => {
+      await expect(
+        client.request(
+          mutation({
+            oldPassword: "1234",
+            newPassword: "1234"
+          })
+        )
+      ).rejects.toThrow("Old password and New password should not be same");
+    });
+  });
+
+  describe("incorrect old password", () => {
+    it('responds with "Old password isn\'t valid"', async () => {
+      await expect(
+        client.request(
+          mutation({
+            oldPassword: "notOldPassword",
+            newPassword: "12345"
+          })
+        )
+      ).rejects.toThrow("Old password isn't valid");
+    });
+  });
```

Comment:
@Kachulio1 you also want to check the happy path, no?

Reply:
After a succefull password change you get back a token a new token don't know how to `expect` that

Reply:
@roschaefer should I also return a message?

Reply:
Never mind found a solution

Reply:
You don't need to return a message.

In order to verify the password change, you could try to login with the old password and expect it to fail. Then you try to login with the new password and expect it to return a token.

Reply:
I actually added a custom expectation 
[jest](https://stackoverflow.com/questions/46103811/how-to-check-for-object-properties-match-for-an-object-using-jest/46103886#46103886)

Reply:
but also that sounds like a good idea